### DISCUSSION TITLE
Getting a message 'Ignoring invalid configuration option passed to Connection', how to solve?

### DISCUSSION THREAD

**Discussion by SwadipSinghoRoy** [sidorares/node-mysql2#1821] [Q&A]
This is actual message:

> **Ignoring invalid configuration option passed to Connection: createDatabaseTable. This is currently a warning, but in future versions of MySQL2, an error will be thrown if you pass an invalid configuration option to a Connection.**

application not respond after getting this message.

I'm using sequelize & as a conncection driver using mysql2, using express-session & as a session store package using express-mysql-session.

Session store configuration code is:

```
const session = require('express-session');
const MySQLStore = require('express-mysql-session')(session);
const mysql = require('mysql2/promise')

const options = {
    host: config.get('db_host'),
    port: config.get('db_port'),
    user: config.get('db_user'),
    password: config.get('db_password'),
    database: config.get('db'),
    createDatabaseTable: false,
    schema: {
        tableName: 'sessions',
        columnNames: {
            session_id: 'sessionId',
            expires: 'sessionExpire',
            data: 'info'
        }
    }
}
const connection = mysql.createPool(options);
const sessionStore = new MySQLStore({}, connection);

session({
    secret: config.get('secret'),
    resave: false,
    saveUninitialized: false,
    store: sessionStore
  })
```
If use more property such as clearExpired, expiration etc. I'm getting warning message for every property. So I remove createDatabaseTable property from options, also application not running. In this situation what should I do?

**Comment by sidorares** [comment]
mysql driver does not support `createDatabaseTable` parameter.

> In this situation what should I do?

Check your app logic. Do you expect driver to create a table for you? You might need to add this manually via something like `connection.query("CREATE TABLE ...`

**Reply by SwadipSinghoRoy** [reply]
That means, first i'll create session table using sql query inside mysql workbench & no extra options pass like: createDatabaseTable, schema etc. But i want to create custom schema.

**Reply by sidorares** [reply]
You can create custom schema using the driver connection, same way as you would do it from workbench

**Reply by SwadipSinghoRoy** [reply]
So that, i need to use `connection.query(....)`. Then no need to create schema inside workbench. If need to set expiration.

**Reply by SwadipSinghoRoy** [reply]
I don't understand how to apply according to your suggestion. If you correction my code & show me how to do that.
